youtube.go

```diff
@@ -409,6 +408,19 @@ func (y *Youtube) log(logText string) {
 	}
 }
 
+func (y *Youtube) Listitags() {
+	if len(y.StreamList) == 0 {
+		fmt.Println("-----no available itag-----")
```

Comment:
should we stick with the same way using function `y.log` to print log?

Reply:
it looks good to me. I agree with you that printing information should belong to UI layer :)